packages/ckeditor5-build-classic/sample/index.html

```diff
@@ -26,9 +26,37 @@ <h2>Sample</h2>
 	<p>You can use this sample to validate whether your <a href="https://ckeditor.com/docs/ckeditor5/latest/installation/advanced/alternative-setups/custom-builds.html">custom build</a> works fine.</p>
 </div>
 
+<div id="inline-editor">
+	<h2>Sample</h2>
+
+	<p>This is an instance of the <a href="https://ckeditor.com/docs/ckeditor5/latest/installation/advanced/alternative-setups/predefined-builds.html#classic-editor">classic editor build</a>.</p>
+
+	<figure class="image">
+		<img src="../tests/manual/sample.jpg" alt="Autumn fields" />
+	</figure>
+
+	<p>You can use this sample to validate whether your <a href="https://ckeditor.com/docs/ckeditor5/latest/installation/advanced/alternative-setups/custom-builds.html">custom build</a> works fine.</p>
+</div>
+
 <script src="../build/ckeditor.js"></script>
 <script>
-	ClassicEditor.create( document.querySelector( '#editor' ) )
+	CKEDITOR.ClassicEditor.create( document.querySelector( '#editor' ), {
+		toolbar: ['RemoveFormat', 'FontFamily', 'FontSize', 'FontColor', 'FontBackgroundColor', 'Underline', 'StrikeThrough',
+		'|', 'heading', 'Alignment',
+		'|', 'bold', 'italic', 'link', 'Strikethrough', 'underline', 'bulletedList', 'numberedList', 'blockQuote',
+		'|', 'SpecialCharacters'],
+	})
+		.then( editor => {
+			window.editor = editor;
+		} )
+		.catch( error => {
+			console.error( 'There was a problem initializing the editor.', error );
```

Comment:
What problem?

Reply:
Can we make it like the one below, where there are no problems? :)
Please add some context to this error state. Why is it needed and why is this the best way to address it?

Reply:
It is a sample used for verifying that build has performed correctly. It was provided by CKEditor team or this logic was moved from old superbuild branch. It is not  used anywhere in our app and do not influence the ckeditor build